src/utilities/dicomJson.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * Converts a naturalized PersonName (json) to a part10 PersonName (string)
+ * @param {object} value json-style person name
+ * @returns {string} part10 dicom representation of PersonName
+ */
+function pnDenaturalize(value) {
+    if (value && typeof value === "object") {
+        return `${value.Alphabetic ?? ""}=${value.Ideographic ?? ""}=${
+            value.Phonetic ?? ""
+        }`.replace(/=*$/, "");
+    }
+    return value;
+}
+
+/**
+ * Overrides toJSON to ensure serialization from part10 to json outputs the
+ * correct structure
+ * @param {string} tag part10 PersonName (PN) tag
+ */
+function pnFromPart10(tag) {
+    // Signal that this value was deserialized from a dcm/part10 file
+    tag.values.__pnDcm = true;
+    // To ensure proper json output, this value will be treated as the json dicom
+    // equivalent when serializing.
+    tag.values.toJSON = function () {
+        const components = this[0]?.split("=");
+        if (components && components.length) {
+            return [
+                {
+                    Alphabetic: components[0],
+                    Ideographic: components[1],
+                    Phonetic: components[2]
```

Comment:
This needs to be a map for all the elements of the array in case the vm>1.   What I would do is check if the base value is a string, and if it is, then convert it into an array and perform the mapping function on the result.  Probably having a function like toArray = (value) => Array.isArray(value) ? value : [value]
You might need to undefined  check as well.